### PR TITLE
PharoLauncher can work with locally created images, not only remote

### PR DESCRIPTION
**Pull request by aryehof**

### CODE REVIEW

File: PharoTour/PharoTour.pillar [hunk 1]
```diff
@@ -171,6 +171,11 @@ download. To download a remote image, select it and click the ==Create
 image== button (located on the top right, next to the ==Refresh template list==
 button).
 
+You can use your own local images with PharoLauncher, in addition to working
+with remote images. To do so, simply ensure that your .image and its associated
+.changes file are placed in a folder (with a name of your choosing) in your
```
Comment (jecisc) [inline PharoTour/PharoTour.pillar]: The name of the .image and .change need to be the same as the folder. I think it would be good to explicate that.

Comment (kilon) [inline PharoTour/PharoTour.pillar]: yeap thats correct , @aryehof you need to add that the image and changes files must be of the exact same name as the folder for this to work, add this commit and i will merge your PR

Comment (aryehof) [inline PharoTour/PharoTour.pillar]: will do

**Comment by kilon** [comment]
well done , merged

**Pull request merged**